_includes/datasets_display.html

```diff
@@ -0,0 +1,72 @@
+
+{% comment %}Only display datasets in current category if there is a current category{% endcomment %}
+{% if current_category %}
+    {% assign current_slug = current_category | slugify %}
```

Comment:
Instead of lines 4-12, why didn't you do `{% assign datasets = site.datasets | where:"category", current_category 
 %}`?

Reply:
It was a legacy from when I was passing that page the slug rather than the full name and so that didn't work, at some point I changed the process but didn't revisit that. But yep, that is much nicer, will make the change.